### PR TITLE
portfolio page (using tailwind) on progress

### PR DESCRIPTION
**Pull request by sushil-adhikari599**

### CODE REVIEW

File: Task/portfolio page/index.html [hunk 1]
```diff
@@ -0,0 +1,1488 @@
...
+          Front End Developer
+        </p>
+        <div class="flex gap-[15px] flex-wrap " id="socialMediaIcons">
+          <img src="images/Facebook.png" />
+          <img src="images/Instagram.png" />
+          <img src="images/Twitter.png" />
+          <img src="images/Linkeding.png" />
+          <img src="images/Youtube.png" />
+          <img src="images/Dribbble.png" />
+        </div>
```
Comment (bipsBro) [inline Task/portfolio page/index.html]:
```suggestion
        <div class="flex gap-[15px] flex-wrap " id="socialMediaIcons">
          <img src="/images/Facebook.png" />
          <img src="/images/Instagram.png" />
          <img src="/images/Twitter.png" />
          <img src="/images/Linkeding.png" />
          <img src="/images/Youtube.png" />
          <img src="/images/Dribbble.png" />
        </div>
```

File: Task/portfolio page/index.html [hunk 2]
```diff
@@ -0,0 +1,1488 @@
...
+    <link
+      href="https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600;700;800;900&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
+    <!-- Applying tailwind via cdn and customizing user own color and font -->
+    <script src="https://cdn.tailwindcss.com"></script>
```
Comment (crisprcoders) [inline Task/portfolio page/index.html]:
```suggestion
    <script src="https://cdn.tailwindcss.com?plugins=forms"></script>
```

File: Task/portfolio page/index.html [hunk 3]
```diff
@@ -0,0 +1,1488 @@
...
+          screens: {
+            //  Add your custom breakpoint here
+            custom: "900px",
+          },
+        },
+      };
+    </script>
+  </head>
+  <body class="flex bg-mainColor">
+    <!-- Left Menu Code -->
```
Comment (crisprcoders) [inline Task/portfolio page/index.html:52]:
```suggestion
      class="flex flex-col justify-between bg-white w-[305px] h-full p-10px gap-y-[25px] max-[767px]:hidden"
```

File: Task/portfolio page/index.html [hunk 4]
```diff
@@ -0,0 +1,1488 @@
...
+                  Address:
+                </p>
+                <p class="font-inter font-[400] text-[15px] text-fontColor2">
+                  35 vhatara, Badda
+                </p>
+              </div>
+            </div>
+          </div>
+          <div
+            class="flex flex-col gap-[30px] p-[25px_25px_17px] bg-white max-[1151px]:w-3/5 max-[885px]:w-4/5 max-[420px]:w-full"
```
Comment (crisprcoders) [inline Task/portfolio page/index.html]:
```suggestion
            class="flex flex-col gap-[30px] p-[25px_25px_17px] bg-white w-full"
```

File: Task/portfolio page/index.html [hunk 5]
```diff
@@ -0,0 +1,1488 @@
...
+          id="contact"
+          class="flex flex-col basis-2/5 gap-[18px] max-[1151px]:flex-grow"
+        >
+          <h2
+            class="font-inter font-[700] text-[32px] text-fontColor1 mb-[32px] max-[1285px]:text-[28px]"
+          >
+            Contact Information
+          </h2>
+          <div
+            class="flex flex-col gap-[30px] p-[25px_25px_17px] bg-white max-[1151px]:w-3/5 max-[885px]:w-4/5 max-[420px]:w-full"
```
Comment (crisprcoders) [inline Task/portfolio page/index.html]:
```suggestion
            class="flex flex-col gap-[30px] p-[25px_25px_17px] bg-white w-full"
```

File: Task/portfolio page/index.html [hunk 6]
```diff
@@ -0,0 +1,1488 @@
...
+                  Telegram:
+                </p>
+                <p class="font-inter font-[400] text-[15px] text-fontColor2">
+                  @username
+                </p>
+              </div>
+            </div>
+          </div>
+          <div
+            class="flex flex-col gap-[30px] p-[25px_25px_17px] bg-white max-[1151px]:w-3/5 max-[885px]:w-4/5 max-[420px]:w-full"
```
Comment (crisprcoders) [inline Task/portfolio page/index.html]:
```suggestion
            class="flex flex-col gap-[30px] p-[25px_25px_17px] bg-white w-full"
```

**Review by crisprcoders — COMMENTED** [review]
Need changes

File: Task/portfolio page/index.html (hunk 5 above)
Comment (crisprcoders) [reply Task/portfolio page/index.html]:
```suggestion
            class="flex flex-col gap-[30px] p-[25px_25px_17px] bg-white w-full"
```

**Pull request merged**